src/extensions/default/CSSCodeHints/main.js

```diff
@@ -204,6 +206,16 @@ define(function (require, exports, module) {
                 $hintObj.text(token.value);
             }
 
+            if (hasColorSwatches) {
+                if (token.color) {
+                    $hintObj.prepend($("<span>")
+                        .addClass("color-swatch")
+                        .css("backgroundColor", token.color));
+                } else {
+                    $hintObj.addClass("left-margin");
+                }
+            }
+
```

Comment:
@MarcelGerber Can you think of any way to make this a function in utils/ColorUtils.js so it can be shared with other code hint providers?